convert-sg-db.py

```diff
@@ -0,0 +1,131 @@
+    if "obsoletedBy" in item:
+        contents["obsoletedBy"] = item["obsoletedBy"]
+    return (item["reference"], contents)
+
+
+def create_spec_whatwg_org(db, template, copyright):
+    content = "<p>The WHATWG works on a number of technologies that are fundamental parts of the web platform. They are organised somewhat arbitrarily based on the preferences of those editing the standard for those technologies.</p>\n\n"
+    content += "<dl>"
+    for workstream in db["workstreams"]:
+        for standard in workstream["standards"]:
+            content += standard_or_idea_to_html(standard)
+    content += "</dl>\n\n<p>The WHATWG also works on a <a href=\"https://idea.whatwg.org/\">number of ideas</a> that aspire to become standards one day.</p>\n\n"
+    content += copyright
+
+    output = template.replace("@TITLE_GOES_HERE@", "Standards")
+    output = output.replace("@CONTENT_GOES_HERE@", content)
+    output = output.replace("<a href=\"https://spec.whatwg.org/\">Standards</a>", "<a>Standards</a>")
+
+    open("spec.whatwg.org/index.html", "w").write(output)
+
+def standard_or_idea_to_html(document):
+    output = "\n <dt><a href=\"{0}\">{1}</a></dt>".format(document["href"], document["name"])
+    return output + "\n <dd>{}</dd>\n".format(commonmark.commonmark(document["description"])[:-1])
+
+
+def create_idea_whatwg_org(db, template, copyright):
+    content = "<p>The WHATWG works on a number of ideas that aspire to become web platform standards.</p>\n\n"
+    content += "<dl>"
+    for idea in db["ideas"]:
+        content += standard_or_idea_to_html(idea)
+    content += "</dl>\n\n"
+    content += copyright
+
+    output = template.replace("@TITLE_GOES_HERE@", "Ideas")
+    output = output.replace("@CONTENT_GOES_HERE@", content)
+
+    open("idea.whatwg.org/index.html", "w").write(output)
+
+
+def create_workstreams(db, template, copyright):
+    content = """<p>This document contains an official <a href="/sg-agreement#steering-group">Steering Group</a>-maintained accounting of the WHATWG workstreams, their responsible editors, and their associated standards.</p>
+
+<p>Note that per the <a href="/workstream-policy">Workstream Policy</a>, the official <a href="/workstream-policy#editor">Editors</a> listed here may have delegated responsibilities and editing duties to deputy editors for any given <a href="/workstream-policy#living-standard">Living Standard</a>. As such, the <a href="/workstream-policy#editor">Editor</a> position listed here is more about "who is the official liaison with the <a href="/sg-agreement#steering-group">Steering Group</a>" and less about "who is editing the document". To get an accounting of who is primarily responsible for day-to-day specification work, you'll be better served by checking the Acknowledgments section of the <a href="/workstream-policy#living-standard">Living Standard</a> in question.</p>"""
+    content += "\n\n<dl>"
```

Comment:
Since you already have a multiline string I'd just insert the `\n\n<dl>` into it?